nym-vpn-core/crates/nym-gateway-directory/src/gateway_client.rs

```diff
@@ -200,7 +198,9 @@ impl GatewayClient {
                     .inspect_err(|err| error!("Failed to parse gateway: {err}"))
                     .ok()
             })
-            .collect();
+            .collect::<Vec<_>>();
+        let skimmed_gateways = self.lookup_skimmed_gateways().await?;
+        append_performance(&mut gateways, skimmed_gateways);
```

Comment:
do we actually need the data from described gateways? if so, what is that so that we could in the future in the `semi-skimmed` endpoint : )

Reply:
It's for the IPR and authenticator addresses. But ideally we shouldn't need either, it should all come from the nym-vpn-api and this is a bandaid until it's added over there :)